packages/cache/src/cache.ts

```diff
@@ -43,6 +43,16 @@ function checkKey(key: string): void {
   }
 }
 
+/**
+ * isFeatureAvailable to check the presence of Actions cache service
+ *
+ * @returns boolean return true if Actions cache service feature is available, otherwise false
+ */
+
+export function isFeatureAvailable(): boolean {
+  return utils.isFeatureAvailable()
```

Comment:
Is it useful to put the function in `utils` instead of just inlining it here?

Reply:
Personally I would opt for keeping the app code simple and pushing complexity into the test code, but it's not a big deal.

You can also simplify the current code like

```
export const isFeatureAvailable = utils.isFeatureAvailable
```

Reply:
it makes sense to keep the app code simple. Inlined the function and moved to test cases to the new file.